npm/prepublish.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const fs = require('fs');
+
+const RefMacroRx = new RegExp('(image:)?(?:(https?:[^\\[]+)|\{([a-z0-9_-]+)\})\\[(|.*?[^\\\\])\\]', 'g');
+
+function markdownify (asciidoc) {
```

Comment:
It's clever, I think we should share this code as part of a library 😉

Reply:
I agree. That would avoid a lot of copy-paste between projects. My only hesitation is that this kind of a quick hack, so putting it on npm might give people the impression that it's a general purpose converter. I suppose we could name it accordingly.

This converter is more comprehensive than the one I used for bespoke-overview since this README is more advanced.

Reply:
> My only hesitation is that this kind of a quick hack, so putting it on npm might give people the impression that it's a general purpose converter. I suppose we could name it accordingly.

I had thought about it but as you said we can name it accordingly and state in the README that this converter is *not* a general purpose converter. But still I think it can be very useful :+1: